pom.xml

```diff
@@ -96,7 +96,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>18.0</version>
+            <version>29.0-jre</version>
```

Comment:
The `-jre` version requires java 8. The `-android` version allows java 7. From the compatibility point of view `-android` would be better. On the other hand we can't stick to old versions all the time.

When merging this.
- For the next release we should have a hint that we require java 8.
- Maybe we should make the required java version more transparent by introducing a mvn enforcer rule for that.